### PR TITLE
better compliance with appstream data standard

### PR DESCRIPTION
**Pull request by proletarius101**
Changed according to
- https://github.com/flathub/flathub/pull/3724#discussion_r1051620485
- https://github.com/flathub/flathub/pull/3724#discussion_r1052666720
- https://github.com/flathub/flathub/pull/3724#discussion_r1052667447

- - -
This change is [Reviewable](https://reviewable.io/reviews/koreader/koreader/9940)

### CODE REVIEW

File: platform/appimage/koreader.appdata.xml [hunk 1]
```diff
@@ -1,28 +1,40 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <component>
-  <id>org.koreader.KOReader</id>
-  <launchable type="desktop-id">org.koreader.KOReader.desktop</launchable>
+  <id>rocks.koreader.KOReader</id>
```
Comment (Frenzie) [inline platform/appimage/koreader.appdata.xml:3]: I assume this might cause some pain to people who use AppImage installation managers?

Comment (Frenzie) [reply platform/appimage/koreader.appdata.xml:3]: I'm certainly not opposed to changing it, just trying to figure out the impact. :-)

Comment (Frenzie) [reply platform/appimage/koreader.appdata.xml:3]: @pazos Okay with you?

**Review by Frenzie — APPROVED** [review]

File: platform/appimage/koreader.appdata.xml (hunk 1 above)
Comment (pazos) [reply platform/appimage/koreader.appdata.xml:3]: I'm fine as long as the android id isn't changed :)

**Pull request merged**